src/styles/blocks/stars.css

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(../../images/star-active.svg) !important;
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
```

Comment:
Avoid using tag names for styling. Instead of using ':nth-child', consider adding a class to the elements you want to style.